docs/topics/wasm/wasm-libraries.md

```diff
@@ -0,0 +1,113 @@
+[//]: # (title: Add dependencies on Kotlin libraries to Kotlin/Wasm project)
+
+You can use the Kotlin standard library (`stdlib`) and test library ([`kotlin.test`](https://kotlinlang.org/api/latest/kotlin.test/))
+in Kotlin/Wasm out of the box. The version of these libraries is the same as the version of the `kotlin-multiplatform` plugin.
+
+Other official Kotlin (`kotlinx`) and multiplatform libraries are not fully supported yet. You can try experimental versions of such libraries
+by adding the Kotlin [experimental repository](https://maven.pkg.jetbrains.space/kotlin/p/wasm/experimental/) to your Gradle project.
+
+> For Kotlin 1.9.0 and later, use the latest available libraries' versions.
+>
+{type="note"}
+
+## Supported Kotlin libraries for Kotlin/Wasm
+
+You can use one of the following repositories to add Kotlin libraries to your project:
+
+* **Maven** Central for `stdlib` and `kotlin.test` libraries:
+
+  ```kotlin
+  // build.gradle.kts
+  repositories { 
+      mavenCentral()
+  }
+  ```
+
+* **Custom Kotlin** repository for other official Kotlin libraries with experimental support:
+
+  ```kotlin
+  // build.gradle.kts
+  repositories {
+      maven {
+          url = uri("https://maven.pkg.jetbrains.space/kotlin/p/wasm/experimental")
+      }
+  }
+  ```
+
+* **Compose dev** repository for Skiko:
+
+  ```kotlin
+  // build.gradle.kts
+  repositories {
+      maven {
+          url = uri("https://maven.pkg.jetbrains.space/public/p/compose/dev/org/jetbrains/skiko/skiko-wasm/")
```

Comment:
I think the url should be just `https://maven.pkg.jetbrains.space/public/p/compose/dev/`